exporter/clickhousetracesexporter/options.go

```diff
@@ -188,6 +203,19 @@ func (opt *Options) AddFlags(flagSet *flag.FlagSet) {
 	}
 }
 
+type stringSliceValue struct {
+	slice *[]string
+}
+
+func (ssv *stringSliceValue) String() string {
+	return fmt.Sprintf("%v", *ssv.slice)
+}
+
+func (ssv *stringSliceValue) Set(value string) error {
+	*ssv.slice = append(*ssv.slice, value)
+	return nil
+}
+
```

Comment:
This is mostly via Google 🙈 — I'm not really familiar with `flag`; is this the appropraite way to get the desired `--datasource foo --datasource bar` behaviour?